internal/transport/http2_server.go

```diff
@@ -1170,9 +1162,9 @@ func (t *http2Server) keepalive() {
 			}
 			if outstandingPing && kpTimeoutLeft <= 0 {
 				if logger.V(logLevel) {
-					logger.Infof("transport: closing server transport due to idleness.")
+					logger.Infof("transport: closing server transport due to keepalive ping not being acked within timeout %v.", t.kp.Time.String())
 				}
-				t.Close()
+				t.Close(fmt.Errorf("keep alive ping not acked within timeout %v", t.kp.Time.String()))
```

Comment:
s/keep alive/keepalive/

Reply:
Also, if you use the `%s` format specifier, you don't have to call the `String()` method.